test/BUILD.gn

```diff
@@ -579,8 +588,19 @@ static_library("browser_tests_runner") {
 }
 
 if (!is_android) {
-test("brave_browser_tests") {
+
+group("brave_browser_tests") {
   testonly = true
+  if (fix_rust_install_name) {
+    deps = [ ":fix_brave_test_install_name_browser_test" ]
```

Comment:
don't see where this is defined? (ditto for `: fix_brave_test_install_name_unit_test`)

Reply:
https://github.com/brave/brave-core/pull/5493/files#diff-ce683b18e7a5718c59d584bb94ec4243R27